server/src/serializers/VoteSerializer.js

```diff
@@ -0,0 +1,12 @@
+class VoteSerializer {
+  static async getSummary(review) {
```

Comment:
since this is a Vote serializer it should take in a vote as an argument. Instead, the related query should happen in the ReviewSerializer, then pass the collection of related Votes to this serializer to calculate the total